Dockerfile

```diff
@@ -14,6 +14,9 @@ RUN addgroup --gid 2000 --system appgroup && \
 
 WORKDIR /app
 
+# Cache breaking
+ENV BUILD_NUMBER ${BUILD_NUMBER:-1_0_0}
+
```

Comment:
Migrated fix across from the template project to ensure we always pull in the latest dependencies